azure-pipelines/build/linux/du/docker/doclient-lite-docker.yml

```diff
@@ -43,6 +43,24 @@ jobs:
     imageVersion: ${{variables.containerImageVersion}}
     stepsTemplate: 'doclient-lite-docker-steps.yml'
 
+- template: ../templates/do-docker-jobs.yml
+  parameters:
+    targetOsArch: 'debian11_arm32'
```

Comment:
For consistency, do you want to add the comment which is in every target name? "# azure pipelines does not support '.' in display names"